aria_data_loaders/aria_data_utils/adt_data_loader.py

```diff
@@ -0,0 +1,565 @@
+    adt_sequences = ADTSequences(adt_path, is_path=(not is_root), verbose=verbose)
+    sequence_index = 10
+    subsequence_index = 0
+    adt_sequences.load_sequence(sequence_index, subsequence_index)
+    if fwd:
+        print("Trying fwd access")
+    else:
+        print("Trying reverse access")
+    iterator = ADTSubsequenceIterator(adt_sequences.data, reverse=(not fwd))
+    data = next(iterator)
+    print(data)
+    all_object_ids = np.unique(data["segmentation"])
+    all_object_ids = all_object_ids[all_object_ids != 0]
+    random_object_id = np.random.choice(all_object_ids)
+    object_info, _ = adt_sequences.get_all_annotated_objects()
+    print("Object info: ", object_info[random_object_id])
+    print(f"Aria pose: {data['pose'].transform_scene_device.matrix()}")
+    if visualize:
+        plt.imshow(data["rgb"])
+        plt.show()
+        plt.imshow(data["segmentation"])
+        plt.show()
+        adt_sequences.data.visualize_image_with_3dbbox(
+            data["rgb"],
+            data["3dbbox"][random_object_id],
+            data["pose"],
+        )
+        adt_sequences.data.visualize_image_with_3dbbox(
+            data["segmentation"],
+            data["3dbbox"][random_object_id],
+            data["pose"],
+        )
+
+
+@click.command()
+@click.option("--adt-path", type=click.Path(exists=True))
+@click.option("--output-path", type=click.Path(exists=True))
+@click.option("--is-root/--not-root", type=bool, default=True, is_flag=True)
+@click.option("--verbose/--not-verbose", type=bool, default=False, is_flag=True)
+@click.option("--visualize/--no-visualize", type=bool, default=True, is_flag=True)
+def get_object_stats(
+    adt_path: str, output_path: str, is_root: bool, verbose: bool, visualize: bool
+):
+    object_pair_list = []
+    object_cat_to_id_count = {}
+    adt_sequences = ADTSequences(adt_path, is_path=(not is_root), verbose=verbose)
+    for sequence_idx in tqdm(range(len(adt_sequences.file_paths))):
+        adt_sequences.load_sequence(sequence_idx)
+        objects, _ = adt_sequences.get_all_annotated_objects()
+        for oid, oinfo in objects.items():
+            if oinfo.category not in object_cat_to_id_count:
+                object_cat_to_id_count[oinfo.category] = 1
+            else:
+                object_cat_to_id_count[oinfo.category] += 1
+            object_pair_list.append((oid, oinfo.name, oinfo.category))
+    output_file = os.path.join(output_path, "adt_object_stats.yaml")
+    with open(output_file, "w") as f:
+        yaml.dump(object_cat_to_id_count, f, default_flow_style=False)
+    print("Saved object stats to: ", output_file)
+    output_file = os.path.join(output_path, "adt_object_pairs.yaml")
+    with open(output_file, "w") as f:
+        yaml.dump(object_pair_list, f, default_flow_style=False)
+    print("Saved object pairs to: ", output_file)
+
+
+if __name__ == "__main__":
+    get_object_stats()
```

Comment:
could you provide some comment line to run this code if you have?

Reply:
Good idea. Will do in the readme.